src/shared/components/data-table/components/table-controls/components/table-actions/components/table-filters/TableFilters.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+    - Takes in the columns where filters can be applied on
+    - Generates the filter values on the fly
+    - Makes use of the popup panel to display the filters
+*/
```

Comment:
This file is empty. Is this here as a placeholder for future work?

Reply:
Yes, it is.